kettle-plugins/mqtt/src/main/java/org/pentaho/di/trans/step/mqtt/MQTTStreamSource.java

```diff
@@ -73,21 +85,32 @@
   };
 
   public MQTTStreamSource( String broker, List<String> topics, int qualityOfService,
-                           MQTTConsumer mqttConsumer ) {
+                           BaseStreamStep mqttConsumer, boolean isSecure, String username,
+                           String password, Map<String, String> sslConfig ) {
     super( mqttConsumer );
     this.broker = broker;
     this.topics = topics;
     this.qos = qualityOfService;
+    this.isSecure = isSecure;
+    this.username = username;
+    this.password = password;
+    this.sslConfig = sslConfig;
   }
 
+  //  public MQTTStreamSource( String broker, List<String> topics, int qualityOfService,
```

Comment:
should we remove this?

Reply:
Absolutely- I noticed that after pushing.  I'll clean it out with the next commit.

Reply:
👍